src/libtriton/api/api.cpp

```diff
@@ -929,7 +929,20 @@ namespace triton {
     return this->symbolic->isOptimizationEnabled(opti);
   }
 
+  bool API::isMemorySymbolic(const triton::arch::MemoryAccess& mem) const{
+	  this->checkSymbolic();
+	  triton::usize symbId = triton::api.getSymbolicMemoryId(mem.getAddress());
```

Comment:
You should use `this->` instead of `triton::api.` What if `mem.getSize()` is more than a `BYTE`?